faf-commons-data/src/main/java/com/faforever/commons/mod/ModReader.java

```diff
@@ -78,6 +78,12 @@ public Mod readModInfo(InputStream inputStream, Path mountBaseDir) {
       mod.setUiOnly(luaValue.get("ui_only").toboolean());
       mod.setIcon(extractIconPath(luaValue));
 
+      // not all mods have and/or need this value
+      var url = luaValue.get("url").toString();
```

Comment:
change this to an explicit type name since var isn't used anywhere else in the repo

Reply:
Processed in https://github.com/FAForever/faf-java-commons/pull/150/commits/778c666dc0fbeee185d518bda74a038003efec69